omim2obo/namespaces.py

```diff
@@ -84,7 +84,10 @@
 WBBT = Namespace('http://purl.obolibrary.org/obo/WBbt_')                      # C. elegans gross anatomy
 EMAPA = Namespace('http://purl.obolibrary.org/obo/EMAPA_')                    # Mouse gross anatomy and development, timed
 XAO = Namespace('http://purl.obolibrary.org/obo/XAO_')                        # Xenopus Anatomy and development
+# MonDO (Monarch Disease Ontology): About the 2 URI prefixes - The one ending in MONDO_ is the cannonical URI and is the
+#   OBO conformant. The one ending in mondo# is an historical URI prefix which is a consequence of OBO format.
 MONDO = Namespace('http://purl.obolibrary.org/obo/MONDO_')                    # Monarch Disease Ontology
```

Comment:
> It would be good to group the 2 mondo namespaces together in the namespaces.py file and add any comment about why both exist.

Done! I made the comment based on my personal notes when I asked Nico about this a few months ago:
> MonDO (Monarch Disease Ontology): About the 2 URI prefixes - The one ending in MONDO_ is the canonical URI and is the OBO conformant. The one ending in mondo# is an historical URI prefix which is a consequence of OBO format.